src/main/java/skye/data/exception/DukeExceptionType.java

```diff
@@ -0,0 +1,74 @@
+package skye.data.exception;
+
+/**
+ * Represents the various types of exceptions that appear in the program.
+ * It contains the error message for each type.
+ */
+public enum DukeExceptionType {
+
+    DEADLINE_NO_DESCRIPTION(
+    "The description of a deadline cannot be empty.\n"
+            + "Correct usage: deadline <description> /by <date>"
+    ),
```

Comment:
I believe wrapped lines should be indented by 8 spaces? 
```suggestion
    DEADLINE_NO_DESCRIPTION(
            "The description of a deadline cannot be empty.\n"
            + "Correct usage: deadline <description> /by <date>"
    ),
```

Reply:
Thank you for pointing this out!